### PR TITLE
Removed noise from Facebook

### PR DESCRIPTION
**Pull request by AdrienFines**
The removeHelpButton function should now remove all links and images.

### CODE REVIEW

File: services/Facebook.filters.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 
 export function removeHelpButtons(document) {
-  const links = document.querySelectorAll('a[href="#"][id*="-markdown-internal-id"]');
-  links.forEach(link => {
-    // Helps buttons do not have text content
-    if (!link.text) {
-      link.remove();
+  const imgs = document.querySelectorAll('img[src*="https://scontent"]');
+  imgs.forEach(img => {
+    if(img.parentNode.tagName == 'A' && !img.parentNode.text){
```
Comment (Ndpnt) [inline services/Facebook.filters.js:5]: You may use an intermediate variable to avoir repeating `img.parentNode` like this:
```js
    const parent = img.parentNode;
    if (parent.tagName == 'A' && !parent.text) {
      parent.remove();
    }
```

Comment (Ndpnt) [reply services/Facebook.filters.js:5]: Also, it misses some spaces before and after parenthesis :).

**Comment by Ndpnt** [comment]
Good job, smart way to fix the issue! 👍 
I hope we will get less email with this. 😄

File: services/Facebook.filters.js (hunk 1 above)
Comment (AdrienFines) [reply services/Facebook.filters.js]: Done thanks !

**Pull request merged**